force-app/classes/test/AssertionsTest.cls

```diff
@@ -126,18 +107,18 @@ private class AssertionsTest {
   }
 
   @isTest
-  static void givenNullList_hasBeenCalledWithParams_callsAssertEquals() {
+  static void givenEmptyList_hasBeenCalledWithParams_callsAssertEquals() {
```

Comment:
question/fix: shouldn't we have a test case for a mix between direct param (`Object`) and argument matcher (`ArgumentMatcher`)?

Reply:
We have one in MethodSpyTest.
What would be the value in AssertionTest ?

Reply:
Oh I missed it sorry, LGTM

Reply:
I have added it also, this way we lock a bit more the API